### DISCUSSION TITLE
بکاپ گرفتن از اطلاعات مهم

### DISCUSSION THREAD

**Discussion by AC-Lover** [Gozargah/Marzban#328] [General]
درود!

اگر می خواهید از اطلاعات مهم مثل دیتابیس که کل دیتا های مهم در اونجا ذخیره میشه بکاپ بگیرید اونم به صورت خودکار می تونید از اسکریپت زیر استفاده کنید

https://github.com/AC-Lover/backup

کار این اسکریپت چیه؟
از تمامی فایل های پنل از جمله دیتابیس یک نسخه کپی را به صورت فایل فشرده (زیپ - zip) از طریق ربات تلگرام به تلگرام میفرسته
تلگرام هم در حال حاضر یک مکان خوبی برای نگه داری فایل ها به صورت دائمی هست
می تونید تنظیم کنید که هر چند دقیقه یک بار یا هر چند ساعت یک بار از اطلاعات شما بکاپ بگیره و بفرسته
و برای پنل مرزبان،هیدیفای و انواع پنل x-ui هستش

**توی لینک بالا آموزش صفر تا صد رو دادم ( هم فارسی هم انگلیسی هم به صورت ویدئو ) و چیزی باقی نمی مونه که اینجا بگم**

و یک نکته ای که هست اینکه با یک ربات میتونید از هزارن سرور بکاپ بگیرید و نیاز به ساخت ربات جدید نیست یکی کافیه

استار فراموش نشه ❤️
موفق باشید 🌸

**Comment by iamtheted** [comment]
درود بر Ac Lover عزیز 
مرسی بابت سکریپت مفید و عالی 
پنل خیلیا رو با این سکریپت نجات دادی
مرسی که هستی 
💙 
تد

**Comment by amircloner** [comment]
https://github.com/iamtheted/Marzban-Backup/tree/main

**Comment by nilofari700** [comment]
سلام ممنون . حالا بک اپ گرفتیم. چطوری ریستور کنم؟
